src/components/Hero/Hero.stories.tsx

```diff
@@ -371,6 +372,48 @@ export const CampaignBackgroundColors: Story = {
   ),
 };
 
+const FallBackExample = () => {
+  const [imageSrc, setImageSrc] = useState("foobar.jpg");
+  const fallbackImageSrc =
+    "https://loremflickr.com/cache/resized/65535_51121136167_40f20af042_z_300_300_nofilter.jpg";
+  // For some reason IIIF image as fallback does not work...
+  // const fallbackImageSrc = "https://iiif.nypl.org/iiif/2/105224/full/!900,900/0/default.jpg";
+
+  return (
+    <Stack spacing="l">
+      <Heading
+        id="campaign-hero-default"
+        size="heading6"
+        text="Campaign Hero with Custom Image component"
```

Comment:
It's not really using a `Custom Image component`.  Rather, it's a more complex method for handling the `imageSrc` prop.  The title should reflect that.

Reply:
Yes, right. My first approach was "custom image" but then the fallback solution worked better. Will update.